src/manufacturer/manufacturerTable.component.test.tsx

```diff
@@ -132,5 +132,7 @@ describe('Manufacturer Table', () => {
     await waitFor(() => {
       expect(screen.getByText('Manufacturer A')).toBeInTheDocument();
     });
+
+    expect(clearFiltersButton).toBeDisabled();
```

Comment:
This assertion is not need in this test. This test used the table row values for the assertion.